EPR.Antivirus/EPR.Antivirus.Application.UnitTests/Clients/SubmissionStatusApiClientTests.cs

```diff
@@ -195,7 +201,21 @@ private static bool IsValidReport(HttpContent? content, FileType fileType, ScanR
             return false;
         }
 
-        return report.AntivirusScanResult == scanResult
-               && report.BlobContainerName == (fileType == FileType.Pom ? PomContainerName : RegistrationContainerName);
+        var expectedBlobContainer = RegistrationContainerName;
+
+        if (fileType == FileType.Pom)
```

Comment:
Create separate tests for each of these conditionals.
Using conditionals in tests make the tests fragile

Reply:
The conditionals are not part of the tests but used for validating results.